discover/analytics-cloud/articles/01-what-is-liferay-analytics-cloud.markdown

```diff
@@ -0,0 +1,124 @@
+analytics are based on static websites---everyone sees the same page content. By
+additionally providing analytics on content articles, Analytics Cloud helps you
+determine the effectiveness of those articles. Asset Analytics also provides
+insight into people's interaction with dynamic personalized content. It
+generates charts on target audiences engage the content.
+
+Liferay Analytics Cloud release 1.0 supports these Asset types from Liferay
+Digital Enterprise:
+
+- Blogs
+- Documents and Media
+- Forms
+
+Regarding user behavior with a form, for example, Analytics Cloud shows the
+number of views and submissions, the percentage of users abandoning the form,
+and the form's average completion time. A line graph shows the number of views
+over time.
+
+![Figure 1: Analytics Cloud shows you how users interact with your forms.](../images/intro-form-behavior-card.png)
+
+As you view Touchpoint and Asset Analytics you can discern whether an action is
+positive or negative. Here are some examples. 
+
+Positive:
+
+- staying active in sessions
+- frequent sign in
+- submitting a form 
+- giving an article five stars
+
+Negative:
+
+- signing out shortly after signing in
+- passing by key strategic content
+- abandoning a form
+- letting a session timeout
+
+The analytics help you determine better ways to relate to people and earn their
+business. For example, if the majority of people in a target industry Segment
+reactive positively to content that describes a new service you're selling, the
+Segment might be interested in similar services. 
+
+Analytics Cloud uncovers issues too. If lots of people abandon a form after
+entering data leading up to a particular field, for example, the field might be
+problematic. If the field's label is ambiguous, make it clear. If filling in the
+field is cumbersome, make it multiple choice. If the field is more trouble than
+its worth, remove it. The point is Analytics Cloud uncovers trouble spots for
+you to address. Identifying and resolving issues makes your websites better and
+helps you achieve business goals. 
+
+Learning how people get to pages and Assets is also useful.
+
+## Path Analytics
+
+People get to content in different ways (paths):
+
+- Site navigation
+- Search within the site
+- Google search
+- Links inside other pages
+- Links within Assets
+
+If you expect a particular navigation sequence to be popular for locating a
+particular Touchpoint (page), for example, check for it in the Touchpoint's Path
+Analytics. If links inside a particular user's blog posts draw users to target
+content, have that user incorporate more key content links in his blog posts. 
+
+Path Analytics includes analytics on device types and browser types that render
+your site content. Knowing the device types and browser types can help you
+determine which ones to test more and optimize user experiences on. 
+
+You've been introduced to Liferay Analytics Cloud and what it offers. You're now
```

Comment:
We can more artfully conclude this, mentioning the benefits rather than the features.